frontend/src/OneBuilding/MineDetail.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Farm and MineDetail is almost the same, a lot of redundant code, the only difference is that the `row` and `iconstyle` comes in different order in the Stylesheet, and the icon it gets. These are literally the same component.

Reply:
Emmmmm, right now those 2 components are quite similar cause currently we only display some resources data. However, with the process moving on, we gonna add more rules to the game, then we might add some different features buttons like 'speed up food generation rate', 'buying gold', so they will be 2 different components in the future. thats why I separated them into 2 components.

Reply:
OK!